src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -74,23 +75,30 @@ public function load(array $configs, ContainerBuilder $container)
             } else {
                 $twigFilesystemLoaderDefinition->addMethodCall('addPath', array($path, $namespace));
             }
+            $container->addResource(new FileExistenceResource($path));
```

Comment:
not needed here as we don't change the container configuration based on the existence of this folder